components/BottomAppBar/src/private/MDCBottomAppBarLayer.m

```diff
@@ -41,28 +41,34 @@ - (CGPathRef)pathWithCutFromRect:(CGRect)rect
                  layoutDirection:(UIUserInterfaceLayoutDirection)layoutDirection {
   UIBezierPath *bottomBarPath = [UIBezierPath bezierPath];
 
+  static CGFloat kStartAngle;
```

Comment:
Can we document the math/reasoning behind these angle calculations?

Reply:
Done